src/routes/balances/balances.controller.ts

```diff
@@ -17,8 +23,8 @@ export class BalancesController {
     @Param('chainId') chainId: string,
     @Param('safeAddress') safeAddress: string,
     @Param('fiatCode') fiatCode: string,
-    @Query('trusted') trusted?: boolean,
-    @Query('exclude_spam') excludeSpam?: boolean,
+    @Query('trusted', new DefaultValuePipe(false)) trusted: boolean,
```

Comment:
I think this change is totally OK, but I was wondering... should we enforce parsing to boolean at this level, as it's [done here](https://github.com/safe-global/safe-client-gateway/blob/8c4dd24d1ac33e12da8deeff49426cd7a74ff66b/src/routes/transactions/transactions.controller.ts#L194) by using `ParseBoolPipe`? 

It's also present in the example for providing default values in [the Nest docs](https://docs.nestjs.com/pipes#providing-defaults).

This is probably something we could address in a separate PR if we think it's needed, as it affects other controllers also. What do you think?

Reply:
Good point. I think we can add the parsing across the controllers that we have 👍